Makefile

```diff
@@ -33,59 +33,15 @@ download-haskell-config: ## Download Cardano Haskell configuration for $NETWORK
 	curl -O --output-dir $(HASKELL_NODE_CONFIG_DIR) $(HASKELL_NODE_CONFIG_SOURCE)/$(NETWORK)/alonzo-genesis.json
 	curl -O --output-dir $(HASKELL_NODE_CONFIG_DIR) $(HASKELL_NODE_CONFIG_SOURCE)/$(NETWORK)/conway-genesis.json
 
-snapshots/$(NETWORK): ## Download snapshots
-	@if [ ! -f "${SNAPSHOTS_FILE}" ]; then echo "SNAPSHOTS_FILE not found: ${SNAPSHOTS_FILE}"; exit 1; fi;
-	mkdir -p $@
-	cat $(SNAPSHOTS_FILE) \
-		| jq -r '.[] | "\(.point) \(.url)"' \
-		| while read p u; do \
-			echo "Fetching $$p.cbor"; \
-			curl --progress-bar -o - $$u | gunzip > $@/$$p.cbor; \
-		done
-
-import-snapshots: snapshots/$(NETWORK) ## Import snapshots for demo
-	@SNAPSHOT_ARGS=""; \
-	CBOR_FILES=$$(find "$^" -maxdepth 1 -name '*.cbor'); \
-	if [ -z "$$CBOR_FILES" ]; then echo "No .cbor files found in $^"; exit 1; fi; \
-	for SNAPSHOT in $(wildcard $^/*.cbor); do \
-		SNAPSHOT_ARGS="$$SNAPSHOT_ARGS --snapshot $$SNAPSHOT"; \
-	done; \
-	cargo run --profile $(BUILD_PROFILE) -- import-ledger-state \
-		--network $(NETWORK) \
-		--ledger-dir "$(LEDGER_DIR)" \
-		$$SNAPSHOT_ARGS
-
-import-headers: ## Import headers from $AMARU_PEER_ADDRESS for demo
-	@if [ ! -f "$(HEADERS_FILE)" ]; then echo "HEADERS_FILE not found: $(HEADERS_FILE)"; exit 1; fi; \
-	HEADERS=$$(jq -r '.[]' $(HEADERS_FILE)); \
-	for HEADER in $$HEADERS; do \
-		cargo run --profile $(BUILD_PROFILE) -- import-headers \
-			--network $(NETWORK) \
-			--chain-dir $(CHAIN_DIR) \
-			--peer-address $(AMARU_PEER_ADDRESS) \
-			--starting-point $$HEADER \
-			--count 2; \
-	done
-
-import-nonces: ## Import nonces for demo
-	@if [ ! -f "$(NONCES_FILE)" ]; then echo "NONCES_FILE not found: $(NONCES_FILE)"; exit 1; fi; \
-	cargo run --profile $(BUILD_PROFILE) -- import-nonces \
-		--network $(NETWORK) \
-		--chain-dir $(CHAIN_DIR) \
-		--at $$(jq -r .at $(NONCES_FILE)) \
-		--active $$(jq -r .active $(NONCES_FILE)) \
-		--candidate $$(jq -r .candidate $(NONCES_FILE)) \
-		--evolving $$(jq -r .evolving $(NONCES_FILE)) \
-		--tail $$(jq -r .tail $(NONCES_FILE))
```

Comment:
Why remove those? I know about the new "bootstrap" command but I don't very much like it because it lacks granularity and default to doing all steps (although it does skip some at times). I often find myself re-importing _just the snapshots_ when working on the ledger; because nonces and headers are usually quite stable.

Reply:
_nevermind_; just saw the commit down the line restoring them.